js/views/stock.js

```diff
@@ -1 +1,29 @@
 // define your StockView here
+function StockView() {
+  $('#update-symbol').on('click', this.lookupSymbol.bind(this));
+  $('#update-shares').on('click', this.updateShares.bind(this));
+}
+
+StockView.prototype = {
+  lookupSymbol: function() {
+    var symbol = $("#symbol-input").val();
+    this.model = new Stock(symbol);
+    this.model.getSymbol(symbol);
+    this.render();
```

Comment:
line 11 here is asynchronous. This is why you have to click the button twice to see the results. `this.render()` is called before the AJAX call completes its round trip to and from the server.